tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -628,7 +636,10 @@ final class BuildRunner extends Runner {
         final io.Process process = await processRunner.processManager.start(
           command,
           workingDirectory: engineSrcDir.path,
-          environment: rbeConfig.environment,
+          environment: {
+            ...rbeConfig.environment,
+            'CLICOLOR_FORCE': '1',
```

Comment:
I wonder if this is correct, that is, if this should be conditionally set _if_ the current environment is an ANSI terminal with color support. 

If it was conditionally set, would we need the parsing and massaging you did above?

Reply:
The behavior most people follow is if you aren't connected to a tty you don't emit ansi escape sequences.  That's the behavior ninja was doing.  I can emulate it.  It's only a problem if someone wants to run et as a subprocess.

Reply:
Done, now we only show the colors if stdout supports it or if it has been overwritten with `CLICOLOR_FORCE`.

Reply:
> If it was conditionally set, would we need the parsing and massaging you did above?

Yes, we still need it because the gcc path fix is a whole different thing that assumed no ansi escape sequences.